core/src/main/java/ai/timefold/solver/core/impl/score/director/AbstractScoreDirector.java

```diff
@@ -343,6 +343,11 @@ public void triggerVariableListeners() {
         variableListenerSupport.triggerVariableListenersInNotificationQueues();
     }
 
+    @Override
+    public void clearVariableListenerEvents() {
+        variableListenerSupport.clearAllVariableListenerEvents();
+    }
```

Comment:
I do not like that we expose this operation on Score Director for everyone to see.
This is a highly specific, single-purpose, _very destructive_ operation.
We should think of a way to limit the exposure; I believe that even the solver developers shouldn't have easy access to this.